tests/test_other.py

```diff
@@ -11180,6 +11180,10 @@ def test_wasmfs_getdents(self):
     self.set_setting('WASMFS')
     self.do_run_in_out_file_test('wasmfs/wasmfs_getdents.c')
 
+  def test_readfile(self):
+    self.set_setting('WASMFS')
+    self.do_run_in_out_file_test(test_file('wasmfs/wasmfs_readfile.c'))
+
```

Comment:
Instead of a new test, as this enables support for preloaded images and audios, perhaps we can enable an existing test? Those might be in the browser suite.

Reply:
@kripken Do you know of any particular test? I looked around for one.

Reply:
I don't know of one offhand, but I would look at the ones with `--use-preload-plugins`, like `test_sdl_image_jpeg` perhaps.